addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -173,8 +171,8 @@ def _generate_efaktur_invoice(self, delimiter):
             eTax['TAHUN_PAJAK'] = move.invoice_date.year
             eTax['TANGGAL_FAKTUR'] = '{0}/{1}/{2}'.format(move.invoice_date.day, move.invoice_date.month, move.invoice_date.year)
             eTax['NPWP'] = invoice_npwp
-            eTax['NAMA'] = move.partner_id.name if eTax['NPWP'] == '000000000000000' else commercial_partner.l10n_id_tax_name or move.partner_id.name
-            eTax['ALAMAT_LENGKAP'] = move.partner_id.contact_address.replace('\n', '') if eTax['NPWP'] == '000000000000000' else commercial_partner.l10n_id_tax_address or street
+            eTax['NAMA'] = etax_name
+            eTax['ALAMAT_LENGKAP'] = commercial_partner.contact_address.replace('\n', '').strip() if eTax['NPWP'] == '000000000000000' else move.partner_id.l10n_id_tax_address or street
```

Comment:
The previous version was correct (missing maybe the strip())
if it's a standard address field, you should take it from partner_id. For l10n_id_tax_address if your partner is child of a company, you should take the one from the company (commercial partner) because I believe the field won't show on the children record.

Reply:
indeed you're right, if it's a children of company, tax address won't show,
thanks